curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-the-date-object-by-building-a-date-formatter/646ae2c40c98e8028dc7072f.md

```diff
@@ -0,0 +1,53 @@
+---
+id: 646ae2c40c98e8028dc7072f
+title: Step 8
+challengeType: 0
+dashedName: step-8
+---
+
+# --description--
+
+Create a HTML `div` element with the class name `divider` that will be later used as an underline for the title.
```

Comment:
Let's remove this step since this is the fourth project in the curriculum. 

By this point, we are supplying the campers with all of the HTML and CSS.

Reply:
removed